insomnia_workspace.json

```diff
@@ -961,6 +961,43 @@
       "settingFollowRedirects": "global",
       "_type": "request"
     },
+    {
```

Comment:
Please remove `qa_getPendingTransactions` from insomnia also

Reply:
And remember to notify QA once this gets merged :)